.circleci/config.yml

```diff
@@ -133,141 +217,114 @@ jobs:
+          <<: *defaults_display_versions
       - restore_cache:
           key: dependency-cache-{{ .Environment.CIRCLE_SHA1 }}
       - run:
-          # This is needed for legacy core tests. Remove this once 'tape' is fully deprecated.
-          name: Install tape, tapes and tap-xunit
-          command: npm install tape tapes tap-xunit
+          name: Install jest
+          command: npm install jest
       - run:
           name: Create dir for test results
           command: mkdir -p ./test/results
       - run:
           name: Execute unit tests
-          command: npm -s run test:xunit
+          command: npm -s run test:junit
       - store_artifacts:
           path: ./test/results
-          prefix: test
+          destination: test
       - store_test_results:
           path: ./test/results
 
   test-coverage:
     executor: default-docker
+    environment:
+      <<: *defaults_environment
     steps:
       - run:
           name: Install general dependencies
-          command: *defaults_Dependencies
+          command: *defaults_docker_Dependencies
       - run:
           name: Install AWS CLI dependencies
           command: *defaults_awsCliDependencies
       - checkout
       - run:
           <<: *defaults_configure_nvm
+      - run:
+          <<: *defaults_display_versions
       - restore_cache:
           key: dependency-cache-{{ .Environment.CIRCLE_SHA1 }}
       - run:
           name: Execute code coverage check
           command: npm -s run test:coverage-check
       - store_artifacts:
           path: coverage
-          prefix: test
-      - store_test_results:
-          path: coverage
-      - run:
-          name: Copy code coverage to SonarQube
-          command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ];
-            then
-                echo "Sending lcov.info to SonarQube..."
-                aws s3 cp coverage/lcov.info $AWS_S3_DIR_SONARQUBE/${CIRCLE_PROJECT_REPONAME}/lcov.info
-            else
-                echo "Not a release (env CIRCLE_BRANCH != 'master'), skipping sending lcov.info to SonarQube."
-            fi
-
-  test-integration:
-    executor: default-machine
-    steps:
-      - checkout
-      - run:
-          <<: *defaults_configure_nvm
-      - restore_cache:
-          key: dependency-cache-{{ .Environment.CIRCLE_SHA1 }}
-      - run:
-          name: Create dir for test results
-          command: mkdir -p ./test/results
-      - run:
-          name: Build and start the docker containers
-          command: |
-            ## This is not needed as we are only doing narrow-integration tests.
-            # docker-compose build 
-
-            ## Lets pull only the Services needed for the Integration Test
-            docker-compose pull mysql kafka
-
-            ## Lets startup only the Services needed for the Integration Test
-            docker-compose up -d mysql kafka
+          destination: test
+      
+# TODO: Uncomment when there are integration tests
```

Comment:
It is okay to remove this commented code since we don't have integration tests for this repo yet or have a story number for the integration test to replace the "TODO".